src/main/java/com/cg_vibely_social_service/configuration/RedisConfiguration.java

```diff
@@ -0,0 +1,100 @@
+package com.cg_vibely_social_service.configuration;
+
+import com.cg_vibely_social_service.utils.LocalDateTimeTypeAdapter;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.data.redis.LettuceClientConfigurationBuilderCustomizer;
+import org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration;
+import org.springframework.cache.annotation.CachingConfigurerSupport;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.redis.cache.RedisCacheConfiguration;
+import org.springframework.data.redis.cache.RedisCacheManager;
+import org.springframework.data.redis.connection.RedisConnectionFactory;
+import org.springframework.data.redis.connection.RedisStandaloneConfiguration;
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
+import org.springframework.data.redis.core.RedisTemplate;
+import org.springframework.data.redis.core.ValueOperations;
+import org.springframework.data.redis.serializer.GenericJackson2JsonRedisSerializer;
+import org.springframework.data.redis.serializer.RedisSerializationContext;
+import org.springframework.data.redis.serializer.StringRedisSerializer;
+
+import java.net.URI;
+import java.time.LocalDateTime;
+
+@Configuration
+@EnableCaching
+@AutoConfigureAfter(RedisAutoConfiguration.class)
+public class RedisConfiguration extends CachingConfigurerSupport {
+
+    private final String redisHost = "localhost";
+
+
+    private final int redisPort = 6379;
+
+
+    @Bean
+    public LettuceConnectionFactory redisConnectionFactory() {
+        RedisStandaloneConfiguration config = new RedisStandaloneConfiguration(redisHost, redisPort);
+        return new LettuceConnectionFactory(config);
+    }
+
+//    @Bean
```

Comment:
If you comment for future features, should use//TODO: ... to declare usage of the block code instead of just comment. If not using anymore, pls remove them!

Reply:
Resolved